<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_in</class>
 <widget class="QWidget" name="Sign_in">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的世界---登录</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/png/ui_icon.ico</normaloff>:/png/ui_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(86, 118, 162);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resource/resource.qrc">:/png/ui_login.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>账号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ui_account">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>12</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border-style: outset;
    border-radius: 8px;
	color: rgb(86, 118, 162);
	border:2px solid rgb(186, 189, 182);</string>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="cursorMoveStyle">
              <enum>Qt::LogicalMoveStyle</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ui_register_account">
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	color: rgb(138, 226, 52);
    }
QPushButton:hover {
	color: rgb(115, 210, 22);
    }
QPushButton:pressed {
	background-color: transparent;
	border: none;
	color: rgb(86, 118, 162);
    }
</string>
             </property>
             <property name="text">
              <string>注册账号</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ui_password">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>12</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">    border-style: outset;
    border-radius: 8px;
	color: rgb(86, 118, 162);
	border:2px solid rgb(186, 189, 182);</string>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ui_forget_password">
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
	color: rgb(138, 226, 52);
    }
QPushButton:hover {
	color: rgb(115, 210, 22);
    }
QPushButton:pressed {
	background-color: transparent;
	border: none;
	color: rgb(86, 118, 162);
    }</string>
             </property>
             <property name="text">
              <string>忘记密码</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="ui_save_password">
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>12</pointsize>
               <underline>true</underline>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">text-decoration: underline;</string>
             </property>
             <property name="text">
              <string>记住账号</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="ui_sign_in">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(149, 187, 209, 255), stop:1 rgba(218, 238, 249, 255));
    border-style: outset;
    border-radius: 8px;
	color:rgb(86, 118, 162);
    }
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 100, 162, 255), stop:1 rgba(61, 140, 201, 255));
    border-style: inset;
    border-radius: 8px;
	color:white;
    }</string>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
  <include location="../../live_client/resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
